linera-views/src/scylla_db.rs

```diff
@@ -231,19 +313,15 @@ impl ScyllaDbStoreInternal {
     }
 
     async fn read_value_internal(
-        store: &ScyllaDbStorePair,
+        store: &ScyllaDbClient,
```

Comment:
As a rule, static methods (or functions) whose main argument is `store: &ScyllaDbClient` should be `&`-methods of `ScyllaDbClient`.

Reply:
Correct, thank you very much.